.circleci/config.yml

```diff
@@ -28,22 +35,72 @@ jobs:
           command: make package
       - deploy:
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              docker login -u $DOCKER_USER -p "$DOCKER_PASS"
-              make docker-push
-              export GCS_KEY_DIR=$(pwd)/keys
-              mkdir -p $GCS_KEY_DIR
-              echo "$GCS_KEY" > $GCS_KEY_DIR/gcs-key.json
-              make publish-packages
-            fi
+            docker login -u $DOCKER_USER -p "$DOCKER_PASS"
```

Comment:
What was the reason we needed to remove the check for being in the master branch?  It seems this would push all branch builds to our package repo which may not be what we want.